Workbooks/DigitalShadows.json

```diff
@@ -0,0 +1,295 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Digital Shadows\n---\n\nWelcome to your new Digital Shadows workbook. Here you can visualize certain aspects of data.\n\nYou can get a list of incidents from past n number of days, bar chart depicting the number of incidents over certain period of time and breakdown of incidents by type as seen in Digital Shadows portal.\n"
```

Comment:
The description is ok for now, however lets get it reviewed later by DS. Can you create a task for the same, so that we do not forget about this.